client/pyroclient/client.py

```diff
@@ -218,12 +218,12 @@ def fetch_detections(self) -> Response:
             timeout=self.timeout,
         )
 
-    def label_sequence(self, sequence_id: int, is_wildfire: bool) -> Response:
+    def label_sequence(self, sequence_id: int, is_wildfire: str) -> Response:
```

Comment:
Should we also include the enum in the client? So that it is not a random string here?